packages/components/package.json

```diff
@@ -29,6 +29,48 @@
   },
   "homepage": "https://github.com/cultureamp/kaizen-design-system#readme",
   "dependencies": {
+    "@kaizen/a11y": "^1.6.22",
+    "@kaizen/brand": "^1.5.4",
+    "@kaizen/brand-moment": "^1.12.55",
+    "@kaizen/button": "^2.0.17",
+    "@kaizen/component-base": "^1.1.3",
+    "@kaizen/date-picker": "^5.17.9",
+    "@kaizen/design-tokens": "^10.2.0",
```

Comment:
Hmm...so if we're exporting `@kaizen/design-tokens` from here, do we therefore remove the peerDep? Or we still need to keep it?

Reply:
Oh good point